x-pack/plugins/lens/public/visualizations/partition/suggestions.ts

```diff
@@ -279,7 +283,7 @@ export function suggestions({
       title: i18n.translate('xpack.lens.pie.waffleSuggestionLabel', {
         defaultMessage: 'As Waffle',
       }),
-      score: state?.shape === PieChartTypes.WAFFLE ? 0.7 : 0.5,
+      score: state?.shape === PieChartTypes.WAFFLE ? 0.7 : 0.4,
```

Comment:
ℹ️ Changing the score here because a FT was failing (for 0.5 it gets higher priority than donut charts) so I reduced it to not break the tests.